### ISSUE TITLE
LiveClient.IsConnection() throws NullReferenceException when called without websocket client instance.

### ISSUE THREAD

**Issue by acidbubbles** [deepgram/deepgram-dotnet-sdk#296]
## What is the current behavior?
`NullReferenceException`

## Steps to reproduce
```c#
var live = new LiveClient();
var _ = live.IsConnected(); // This throws because _clientWebSocket can be null
```

## Expected behavior
`false`

## Please tell us about your environment
Not related

## Other information
I would recommend you enable `<Nullable>enable</Nullable>` on the project, this one would have been easily detected :)

Also, I can receive that same error _after_ calling Connect. Probably because of KeepAlive but I didn't investigate more.

**Comment by dvonthenen** [comment]
Hi @acidbubbles 

Nice catch. Will fix this when I get back to .NET.

**Comment by dvonthenen** [comment]
Will be available in the 4.1 release.
